addons/binding/org.openhab.binding.hyperion/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hyperion" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="serverV1">
+		<label>Hyperion Server</label>
+		<description>This is a Hyperion server</description>
+		<channels>
+			<channel id="brightness" typeId="brightness" />
```

Comment:
please have in mind that the `color` channel with `Color` item does already provide a `brightness` property. The `Color` item actually does HSB control, so hue, saturation and brightness are already present. The Color item also provides `brightness only` control when the user defines a `Dimmer` item for that channel (and On/Off control when using `Switch` item. You may drop the brightness channel here (and in the ng server definition).

Reply:
As mentioned, this channel is still required